### DISCUSSION TITLE
Lines of code removed and added doesn't work at all

### DISCUSSION THREAD

**Discussion by AlphaNecron** [lowlighter/metrics#1500] [Q&A]
![image](https://github.com/lowlighter/metrics/assets/57827456/a1cb8cc0-7699-4f3f-9fca-f1037774bd51)

My CI workflow: https://github.com/AlphaNecron/AlphaNecron/blob/master/.github/workflows/readme.yml

Token scopes:
![image](https://github.com/lowlighter/metrics/assets/57827456/2b02264e-9b67-4151-a15e-cd57412a40da)

It was working a few days ago.

**Comment by lowlighter** [comment]
Hi 👋 !

Using the new option `plugin_lines_delay` from #1530 that was added in last version may fix the issue.

The API endpoints used by this plugin seems to use some cache mechanism, which sends back zeroed data if they're missing, and are only refreshed after a few seconds when you query back the data

So something like `plugin_lines_delay: 30` could fix this

**Comment by AlphaNecron** [comment]
Works now, thanks a lot.
